ui/src/routes/Lessons/Lessons.js

```diff
@@ -10,6 +12,23 @@ import '../../styles/buttons.css'
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Lessons extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```

Comment:
Do you need to define `skipLinks` here? Looks like it's not used, and I thought it was already defined in the state of the new `SkipLinks` component?